exercises/protein-translation/protein_translation_test.go

```diff
@@ -0,0 +1,63 @@
+package protein
+
+import (
+	"testing"
+)
+
+const targetTestVersion = 1
+
+type testCase struct {
+	input    string
+	expected string
+}
+
+var codonTestCases = []testCase{
+	{"AUG", "Methionine"},
+	{"UUU", "Phenylalanine"},
+	{"UUC", "Phenylalanine"},
+	{"UUA", "Leucine"},
+	{"UUG", "Leucine"},
+	{"UCU", "Serine"},
+	{"UCC", "Serine"},
+	{"UCA", "Serine"},
+	{"UCG", "Serine"},
+	{"UAU", "Tyrosine"},
+	{"UAC", "Tyrosine"},
+	{"UGU", "Cysteine"},
+	{"UGC", "Cysteine"},
+	{"UGG", "Tryptophan"},
+	{"UAA", "STOP"},
+	{"UAG", "STOP"},
+	{"UGA", "STOP"},
+}
+
+var proteinTestCases = []testCase{
```

Comment:
Small question because I am curious, not asking for anything to change: Did you get these test cases from any other track that implemented this exercise, or from elsewhere?